src/components/Groups/Group.tsx

```diff
@@ -49,6 +50,7 @@ function extractData(
           chainIdToNetworkName[balanceEntry.wallet.chainId.toString()],
         symbol: balanceEntry.balance.symbol,
         tokenName: balanceEntry.balance.name,
+        decimals: balanceEntry.balance.decimals,
         quantity: convertWeiToQuantity(
           balanceEntry.balance.balance,
           balanceEntry.balance.decimals,
```

Comment:
Do we need that after all these changes?

Reply:
No, i guess we dont need it anymore because its calculated inside TokenRow component. i will remove this